association_backend/association/domain/entities/subscriptions.py

```diff
@@ -0,0 +1,80 @@
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+import pydantic
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, Table
+from sqlalchemy.orm import registry, relationship
+
+
+class UserData(pydantic.BaseModel):
+    email: str
+    user_id: int
+
+
+class SubscriptionState(str, Enum):
+    PENDING = "pending"
+    ACTIVE = "active"
+    EXPIRED = "expired"
+    NOT_CREATED = "not-created"
+
+    def __str__(self) -> str:
+        return str.__str__(self)
+
+
+@dataclass
+class Subscription:
+    user_id: int
+    creation_date: datetime
+    state: SubscriptionState
+    # is_for_life: bool
+    stripe_session_id: Optional[str] = ""
+    stripe_id: Optional[str] = ""
```

Comment:
@marcoacierno do you think we have to rename it in stripe_subscription_id?

Reply:
yes please, so it's more clear!

Reply:
ok done